src/platform/testing/e2e/cypress/support/commands/login.js

```diff
@@ -0,0 +1,63 @@
+import { VA_FORM_IDS } from 'platform/forms/constants';
+
+/* eslint-disable camelcase */
+const mockUser = {
+  data: {
+    attributes: {
+      profile: {
+        sign_in: {
+          service_name: 'idme',
+        },
+        email: 'fake@fake.com',
+        loa: { current: 3 },
+        first_name: 'Jane',
+        middle_name: '',
+        last_name: 'Doe',
+        gender: 'F',
+        birth_date: '1985-01-01',
+        verified: true,
+      },
+      veteran_status: {
+        status: 'OK',
+        is_veteran: true,
+        served_in_military: true,
+      },
+      in_progress_forms: [
+        {
+          form: VA_FORM_IDS.FORM_10_10EZ,
+          metadata: {},
+        },
+      ],
+      prefills_available: [VA_FORM_IDS.FORM_21_526EZ],
+      services: [
+        'facilities',
+        'hca',
+        'edu-benefits',
+        'evss-claims',
+        'form526',
+        'user-profile',
+        'health-records',
+        'rx',
+        'messaging',
+      ],
+      va_profile: {
+        status: 'OK',
+        birth_date: '19511118',
+        family_name: 'Hunter',
+        gender: 'M',
+        given_names: ['Julio', 'E'],
+        active_status: 'active',
+      },
+    },
+  },
+  meta: { errors: null },
+};
+/* eslint-enable camelcase */
+
+/**
+ * Simulates a logged in session.
+ */
+Cypress.Commands.add('login', () => {
+  window.localStorage.setItem('hasSession', true);
+  cy.server().route('GET', '/v0/user', mockUser);
```

Comment:
A session is mocked with only this user for now. We'll need to expand on this as more apps require a more customizable user/session.

Reply:
So in the future when more `mockUsers` are used, will we add a parameter to this command?

Reply:
Ideally we'd have integration with Pact and just pull the corresponding user object based on a string descriptor. Any meaningful differences should be captured in a contract anyway.

Reply:
That integration might result in this having a parameter or just another command altogether. I'll have to see where I end up going with it.